cc_licenses/templates/all_licenses.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
```

Comment:
Legal Code section for unported licenses are missing a closing `div`. Sections that are meant to be distinct have the legal code section styles applying to them (they are included in the legal code section).

Reply:
@Audiosutras can you expand on this? The template being commented on has no `div` tags to close.

Reply:
@dpoirier 

![image](https://user-images.githubusercontent.com/37882933/98159238-3658e080-1eaa-11eb-83a2-a735030169cf.png)

Reply:
Starting at `View Legal Code as plain text` and down, these sections should be outside of the legal code container.

Reply:
Sorry about the inaccurate comment location.

Reply:
It looks like there's a missing `</div>` in `legalcode_30_ported_license.html`, I'll fix that.

Reply:
looks much better.